models/response.go

```diff
@@ -6,10 +6,11 @@ type ResponseError struct {
 }
 
 type Response struct {
```

Comment:
We were considering utilizing composition in responses as we wont necessarily want the React component to handle all of the file artifacts. 
i.e. 
`type ScreenShotResponse struct {`
`    Response`
`    ScreenShot []byte`
}
Thoughts?

Reply:
Is the way this current PR is laid out such that our API filters through all of the file artifacts and only retrieves the one that matches the {fileName} specified in the request?

Reply:
That makes more sense than having specific responses.

Reply:
Filenames + ticket IDs are unique, yup! So there's two endpoints - one that gets the list of all filenames associated with a ticket, and another to download a specific file. The reasoning for returning the files individually instead of one big response -
1) browser can cache the files
2) no base64 overhead for encoding then decoding
3) waiting for / downloading the whole response is going to take a long time with big files, versus hitting the cache and only downloading new stuff

Reply:
Because we're always writing `screenshotFull.png` for now, for every ticket you could pull up `http://localhost:8080/api/tickets/{ticket id here}/artifacts/screenshotFull.png`, and get the screenshot as an image in the browser. It would be less work to just hard code that if we don't want to deal with the whole multiple files thing on the frontend for now - show a loading spinner until the GET endpoint for the ticket shows that `processed` is `true`, then add a new image element with the `screenshotFull.png` URL.